conf/nebula-storaged-listener.conf.production

```diff
@@ -0,0 +1,53 @@
+########## nebula-storaged-listener ###########
+########## basics ##########
+# Whether to run as a daemon process
+--daemonize=true
+# The file to host the process id
+--pid_file=pids_listener/nebula-storaged.pid
+
+########## logging ##########
+# The directory to host logging files, which must already exists
+--log_dir=logs_listener
+# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+--minloglevel=0
+# Verbose log level, 1, 2, 3, 4, the higher of the level, the more verbose of the logging
+--v=0
+# Maximum seconds to buffer the log messages
+--logbufsecs=0
+
+########## networking ##########
+# Meta server address
+--meta_server_addrs=192.168.2.1:45500
+# Local ip
+--local_ip=192.168.2.4
+# Storage daemon listening port
+--port=44510
+# HTTP service ip
+--ws_ip=192.168.2.4
+# HTTP service port
+--ws_http_port=12021
+# HTTP2 service port
+--ws_h2_port=12031
+
+########## storage ##########
+--listener_path=data/listener
+# The type of part manager, [memory | meta]
+--part_man_type=memory
+# The default reserved bytes for one batch operation
+--rocksdb_batch_size=4096
+# The default block cache size used in BlockBasedTable.
+# The unit is MB.
+--rocksdb_block_cache=4
+# The type of storage engine, `rocksdb', `memory', etc.
+--engine_type=rocksdb
+# The type of part, `simple', `consensus'...
+--part_type=simple
+
+############## rocksdb Options ##############
+--rocksdb_disable_wal=true
+# rocksdb DBOptions in json, each name and value of option is a string, given as "option_name":"option_value" separated by comma
+--rocksdb_db_options={}
+# rocksdb ColumnFamilyOptions in json, each name and value of option is string, given as "option_name":"option_value" separated by comma
+--rocksdb_column_family_options={"write_buffer_size":"67108864","max_write_buffer_number":"4","max_bytes_for_level_base":"268435456"}
+# rocksdb BlockBasedTableOptions in json, each name and value of option is string, given as "option_name":"option_value" separated by comma
+--rocksdb_block_based_table_options={"block_size":"4096"}
```

Comment:
These could be deleted as well, actually rocksdb is not used in listener for now.